test/Turn-test.js

```diff
@@ -22,10 +22,55 @@ describe('Turn', function() {
         expect(turn.guess).to.equal('octopus');
     })
 
-    it('should store the card that is being answered')
-    it('should take in and return a user/s guess')
-    it('should return the card that was played')
-    it('should tell us if the user guess matches the answer on the card')
-    it('should throw a message if the guess was correct')
-    it('should throw a message if guess was incorrect')
+    it('should store the card that is currently being played', function () {
+        const card1 = new Card();
+        const turn = new Turn('octopus', card1);
+
+        expect(turn.playedCard).to.equal(card1);
+    })
+
+    it('should take in and return a user/s guess', function() {
+        const card2 = new Card(1, 'What allows you to define a set of related information using key-value pairs?', ['object', 'array', 'function'], 'object' );
+        const turn = new Turn('octopus', card2);
+        
+        turn.returnGuess();
+
+        expect(turn.returnGuess()).to.equal('octopus');
+    })
+
+    it('should return the card that was being played', function() {
+        const card3 = new Card(1, 'What allows you to define a set of related information using key-value pairs?', ['object', 'array', 'function'], 'object' );
+        const turn = new Turn('turtle', card3);
+
+        turn.returnCard();
+
+        expect(turn.returnCard()).to.equal(card3);
+    })
+
+    it('should tell us if the user guess matches the answer on the card', function() {
```

Comment:
Writing these cards was fun, nice work making sure you understand what the test should be checking for.